velox/functions/sparksql/RegexFunctions.cpp

```diff
@@ -69,7 +203,7 @@ std::shared_ptr<exec::VectorFunction> makeRLike(
     const core::QueryConfig& config) {
   // Return any errors from re2Search() first.
   auto result = makeRe2Search(name, inputArgs, config);
-  ensureRegexIsCompatible("RLIKE", inputArgs[1].constantValue);
+  ensureRegexIsConstantAndCompatible("RLIKE", inputArgs[1].constantValue);
```

Comment:
Function name here is all caps but somewhere else I saw it all lower case. Should it be lower case everywhere?

Reply:
I made changes and made all function names upper case to match other functions found in the file. Not sure what the standard is but changed for consistency.